kpatch-build/kpatch-build

```diff
@@ -554,7 +554,10 @@ if [[ $CHANGED -eq 0 ]]; then
 fi
 
 echo -n "Patched objects:"
-for i in "${!objnames[@]}"; do echo -n " $(basename $i)"; done
+for i in $(echo "${objnames[@]}" | tr ' ' '\n' | sort -u | tr '\n' ' ')
```

Comment:
@jpoimboe If different objects in one module have been changed, there will be duplicates.